### PR TITLE
Make chain index watcher list configurable.

### PR DESCRIPTION
**Pull request by raduom**
Pre-submit checklist:
- Branch
    - [x] Commit sequence broadly makes sense
    - [x] Key commits have useful messages
    - [ ] Relevant tickets are mentioned in commit messages
- PR
    - [x] Self-reviewed the diff
    - [x] Useful pull request description
    - [x] Reviewer requested
- If you updated any cabal files or added Haskell packages:
    - [x] `nix-shell shell.nix --run updateMaterialized` to update the materialized Nix files
    - [ ] Update `hie-*.yaml` files if needed
- If you changed any Haskell files:
    - [x] `nix-shell shell.nix --run fix-stylish-haskell` to fix any formatting issues
- If you changed any Purescript files:
    - [ ] `nix-shell shell.nix --run fix-purty` to fix any formatting issues

Pre-merge checklist:
- [ ] Someone approved it
- [ ] Commits have useful messages
- [ ] Review clarifications made it into the code
- [ ] History is moderately tidy; or going to squash-merge

### CODE REVIEW

File: pre-stylish.diff [hunk 1]
```diff
@@ -0,0 +1,66 @@
+diff --git a/plutus-pab-client/pab-demo-scripts.nix b/plutus-pab-client/pab-demo-scripts.nix
```
Comment (j-mueller) [inline pre-stylish.diff:1]: Delete (I know it's generated by stylish-haskell somehow, it happens on my machine too)

**Review by j-mueller — APPROVED** [review]
LGTM apart from the superfluous file. I'm assuming the part where the chain index actually uses the watched addresses comes in the next PR...

File: pre-stylish.diff (hunk 1 above)
Comment (raduom) [reply pre-stylish.diff:1]: Glad to :)

**Comment by raduom** [comment]
> LGTM apart from the superfluous file. I'm assuming the part where the chain index actually uses the watched addresses comes in the next PR...

Yes. That is the plan.

**Pull request merged**